BeatSpeedrun/Source/Controllers/ModFlowCoordinator.cs

```diff
@@ -0,0 +1,41 @@
+﻿using BeatSaberMarkupLanguage;
+using BeatSpeedrun.Source.Views;
+using HMUI;
+using Zenject;
+
+namespace BeatSpeedrun.Source.Controllers
+{
+    internal class ModFlowCoordinator : FlowCoordinator
+    {
+        private MainFlowCoordinator _mainFlowCoordinator;
+        private MainSettingsView _mainSettingsView;
+        private FloatingTimerViewController _floatingTimerViewController;
+
+        // Normal constructor with no Inject attribute would cause MainSettingsView to be null!
```

Comment:
Can we use the `Inject` attribute for member variables like `LeaderboardMainViewController`?

Reply:
Yes, but constructor/method injection is recommended.
Reference : https://github.com/modesttree/Zenject#injection:~:text=Best%20practice%20is%20to%20prefer%20constructor/method%20injection%20compared%20to%20field/property%20injection.
https://github.com/modesttree/Zenject#injection:~:text=Inject%20methods%20are%20the%20recommended%20approach%20for%20MonoBehaviours%2C%20since%20MonoBehaviours%20cannot%20have%20constructors.